examples/js/controls/OrbitControls.js

```diff
@@ -254,11 +244,9 @@ THREE.OrbitControls = function ( object, domElement ) {
 	var EPS = 0.000001;
 
 	// current position in spherical coordinates
-	var theta;
-	var phi;
+	var spherical = new THREE.Spherical();
+	var sphericalDelta = new THREE.Spherical();
```

Comment:
I think we should just keep `phiDelta` and `thetaDelta` for reasons noted below. `spherical` is OK.